src/functionalTest/resources/conf/routes/handlers/httpRepoConnectorHandler.conf

```diff
@@ -28,7 +28,7 @@ clientDestination {
   domain = localhost
 
   ## Port on which the host listens, e.g. 8080, 3001, etc.
-  port = ${test.wiremock.mockRepository.port}
+  port = 80
```

Comment:
Why static port here?
It may be problematic to run this test in CI.

Reply:
I intend to remove all Knot.x default configuration files from functional tests. Instead I want to attach teh Stack package folder with default configs we use to build distribution package. All scenario specific settings are moved to separate files - in this case we have `common/templating/routing.conf`. In the future we can remove `conf` filder at all.